src/assets/data/organizers.ts

```diff
@@ -0,0 +1,184 @@
+import { Person } from "src/app/core/models/organizer.model";
```

Comment:
Use relative path and single quotes:
```suggestion
import { Person } from '../../app/core/models/organizer.model';
```